easybuild/easyblocks/w/wps.py

```diff
@@ -67,8 +67,8 @@ def __init__(self, *args, **kwargs):
     @staticmethod
     def extra_options():
         testdata_urls = [
-            "http://www.mmm.ucar.edu/wrf/src/data/avn_data.tar.gz",
-            "http://www.mmm.ucar.edu/wrf/src/wps_files/geog.tar.gz",  # 697MB download, 16GB unpacked!
+            "http://www2.mmm.ucar.edu/wrf/src/data/avn_data.tar.gz",
+            "http://www2.mmm.ucar.edu/wrf/src/wps_files/geog_complete.tar.bz2",  # 2.3GB download!
```

Comment:
@boegel Trying to do this, I noticed that the URLs are defined in `extra_options`, which is a static method and thus doesn't have access to `self.version`. I'd appreciate any suggestions how to solve this.